sbnci/scripts/pfpslicevalidation.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+## Allows for different naming structure in cfg file
+export ref_sbndcode_ana_hist=$(echo ${ref_pfpslice_hist})
```

Comment:
I think here 
`export ref_sbndcode_ana_hist=${ref_pfpslice_hist}`
should just work

Reply:
Very true, I have updated!